functions/index.js

```diff
@@ -255,91 +308,73 @@ const updateUser = (req, res) => {
         });
 };
 
-// User Post
 
-const newPost = (req, res) => {
-    const Post = {
-        username: req.user.username,
-        post: req.body.post,
-        tags: req.body.tags,
-        canLikes: req.body.canLikes,
-        canUnlike: req.body.canUnlike,
+ 
 
+// User Post
+// funciona
+const newPost = (req, res) => {
+    const Postuser = {
+        username: req.body.username ||"",
+        post: req.body.post ||"",
+        tag: req.body.tag ||"",
+        canLikes: req.body.canLikes ||"",
+        canUnlike: req.body.canUnlike ||"",
     };
```

Comment:
It would be nice if we have two more parameters to store the dates, for example: createdAt, updatedAt

With those we would know more information about the posts.

We need to potentially change the contract to add those consistently